mysql-test/suite/galera/r/MW-362.result

```diff
@@ -7,7 +7,7 @@ RESET BINARY LOGS AND GTIDS;
 SET SESSION WSREP_ON=ON;
 SHOW BINARY LOGS;
 Log_name	File_size	Encrypted
-mysqld-bin.000001	157	No
```

Comment:
Please check what caused this shift in binary log position by 1 byte.

Reply:
It is related to the introduction of the new event `GTID_TAGGED_LOG_EVENT`. Now `FORMAT_DESCRIPTION_EVENT` header is 1 byte longer.

(c1fcd5f1b73709094e86474e2406ed89b83109e2)